watertap/core/zero_order_costing.py

```diff
@@ -2569,10 +2633,17 @@ def _general_power_law_form(blk, A, B, sizing_term, factor=None):
         )
 
         expr = pyo.units.convert(
-            A * pyo.units.convert(sizing_term, to_units=pyo.units.dimensionless) ** B,
+            A
```

Comment:
This is different than the way we did it in WT3.
In WT3 for models with multiple units, the cost is essentially:

``C = (A * X ** b) * num_units``

Where `X` is calculated with the full flow entering the unit. I am not sure if the idea in WT3 was that there would be one unit offline (i.e. not in parallel), so the flow would be flowing through one unit at a time...

But this is the proper calculation for having parallel units. Because deviates from how it was implemented in WT3, the validation will still be off, but that is OK.